tests/regression/lp-1867193/task.yaml

```diff
@@ -0,0 +1,24 @@
+summary: certain refresh sequence on the maas snap breaks the layout system
+systems: [ubuntu-18.04-64] # tight coupling with container guest
+prepare: |
+    snap install --candidate lxd
+    snap set lxd waitready.timeout=240
+    lxd waitready
+    lxd init --auto
+    lxc launch --quiet "ubuntu:18.04" bionic
+    lxc exec bionic -- apt autoremove --purge -y snapd ubuntu-core-launcher
+    lxc exec bionic -- apt update
+    lxc exec bionic -- mkdir -p "$GOHOME"
+    lxc file push --quiet "$GOHOME"/snapd_*.deb "bionic/$GOHOME/"
```

Comment:
Nitpick, the target dir on bionic doesn't need to be "$GOHOME", does it? It looks slightly confusing.

Reply:
It doesn't but I copied this code from another test.

I can remove it from all the copies I guess.